addon/server.js

```diff
@@ -167,7 +130,7 @@ export default class Server {
     let hasFactories = this._hasModulesOfType(config, 'factories');
     let hasDefaultScenario = config.scenarios && config.scenarios.hasOwnProperty('default');
 
-    this.pretender = this.pretender || createPretender(this);
+    this.interceptor = this.interceptor || createInterceptor(this);
```

Comment:
Fixed: `createPretender` is now setting pretender on `Server`.

https://github.com/mfazekas/ember-cli-mirage/blob/express/addon/interceptors/create-pretender.js#L48